config/ibm_jre.yml

```diff
@@ -0,0 +1,25 @@
+# Cloud Foundry Java Buildpack
+# Copyright 2017 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Configuration for JRE repository
+---
+jre:
+  version: 1.8.+
+  repository_root: "https://public.dhe.ibm.com/ibmdl/export/pub/systems/cloud/runtimes/java/meta/jre/linux/x86_64/"
+  heap_ratio: 0.75
+jvmkill_agent:
+  version: 1.+
+  repository_root: "https://raw.githubusercontent.com/bharathappali/jvmkill/jvmkill-ibmagent/killagent/"
```

Comment:
This needs to be hosted in a bit more permanent of a place than just a GitHub repository.  Because that repository can be deleted at any moment, breaking existing users, this should be hosted somewhere a little more canonical.  I'd prefer it it was hosted at or near the other repository, i.e. some place with an IBM domain.

I do understand that it's only temporary, but there will be users that will see this temporary dependency.